tests/test_main.py

```diff
@@ -440,6 +440,43 @@ def checkpoint_never_exists(path):
         assert not native_exists(os.path.join(".dvc", "tmp", env.DVC_CHECKPOINT))
 
 
+def test_log_image_np_array(tmp_dir):
+    import numpy as np
+    from PIL import Image as ImagePIL
+
+    live = Live()
+    image_data = np.random.randint(0, 255, (100, 100, 3)).astype(np.uint8)
+    live.log_image("foo.png", image_data)
+    live.end()
+
+    plot_file = os.path.join(tmp_dir, live.plots_dir, "images", "foo.png")
+    assert os.path.exists(plot_file)
```

Comment:
This part is already tested in:

https://github.com/iterative/dvclive/blob/1cb31aa2f906396c1a32f1e0bd548d49af851182/tests/plots/test_image.py#L25-L31

The  `assert np.array_equal(val, image_data)` could be added to that test